docs/sources/clients/promtail/stages/logfmt.md

```diff
@@ -0,0 +1,86 @@
+---
+title: logfmt
```

Comment:
Do you mind adding `logfmt` to the list of existing stages in the stages page? (this one: https://grafana.com/docs/loki/latest/clients/promtail/pipelines/)